app/views/idv/in_person/address.html.erb

```diff
@@ -33,7 +33,7 @@
   <%= render ValidatedFieldComponent.new(
         form: f,
         input_html: { value: pii[:address2] },
-        label: t('in_person_proofing.form.address.address2'),
+        label: t('idv.form.address2_optional'),
```

Comment:
Make sure all usages in the `*_spec.rb` files also get updated.

Reply:
> Make sure all usages in the `*_spec.rb` files also get updated.

Thanks, updated in 1c20cf414.